decidim-admin/spec/shared/manage_process_steps_examples.rb

```diff
@@ -97,7 +98,7 @@
     let!(:process_step2) { create(:participatory_process_step, participatory_process: participatory_process) }
 
     before do
-      visit decidim_admin.participatory_process_path(participatory_process)
+      visit current_path
```

Comment:
Oh, since this is a shared example, `current_path` is being defined in the actual example. Why is this a shared example though? Can't see the reason.

Maybe you should define a `current_path` in the shared example and make it blow up with an exception telling the consumer to implement it? Also, `current_path` might not be the best name. What about `resource_path`?